sympy/core/tests/test_args.py

```diff
@@ -1458,16 +1459,22 @@ def test_sympy__stats__joint_rv__JointDistribution():
 
 def test_sympy__stats__joint_rv_types__MultivariateNormalDistribution():
     from sympy.stats.joint_rv_types import MultivariateNormalDistribution
-    assert _test_args(MultivariateNormalDistribution([0, 1], [[1, 0],[0, 1]]))
+    assert _test_args(
```

Comment:
I think that the original test should pass  when the arguments are internally converted to immutable matrices before the `Basic`-object is created. Maybe that could be done in `JointDistribution.__new__`.